### PR TITLE
Dev/jpl/extensibility

### PR DESCRIPTION
**Pull request by jeanplevesque**
GitHub Issue: #
https://github.com/nventive/BiometryService/issues/48

## Proposed Changes

- Feature

## What is the current behavior?

- We can't create custom implementations of `IBiometryService` because the ctor of `BiometryCapabilities` is not public.
- There is no implementation of `IBiometryService` for platforms that don't support biometry.

## What is the new behavior?

- The ctor of `BiometryCapabilities` is now public.
- There is `UnsupportedBiometryService` for platforms that don't support biometry.

## Checklist

Please check if your PR fulfills the following requirements:

- [x] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [x] Associated with an issue

### CODE REVIEW

**Review by takla21 — APPROVED** [review]

**Comment by Iguins** [comment]
Thanks!

**Comment by MaximeDion-Work** [comment]
So the point is that by default call to features on the service throw error if not overwritten first?

File: src/BiometryService/BiometryCapabilities.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace BiometryService
 		/// </summary>
 		public bool IsSupported => BiometryType != BiometryType.None;
 
-		internal BiometryCapabilities(BiometryType biometryType, bool biometryIsEnabled, bool passcodeIsSet)
+		public BiometryCapabilities(BiometryType biometryType, bool biometryIsEnabled, bool passcodeIsSet)
```
Comment (Iguins) [inline src/BiometryService/BiometryCapabilities.cs:35]: Did not think about custom implementation!

**Review by Iguins — APPROVED** [review]

**Comment by jeanplevesque** [comment]
> So the point is that by default call to features on the service throw error if not overwritten first?

@MaximeDion-Work 
Some platforms (like our Tests Project) don't support any kind of biometry. This implementation can be used for that platform because it explicitly says that biometry features are not supported.

Before this change, there was 0 implementation for tests projects.

**Pull request merged**